incubator/spark-history-server/charts/nfs/templates/nfs-server-rc.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.enableExampleNFS }}
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: {{ .Values.nfsServerServiceName }}
+  namespace: {{ .Values.namespace }}
+spec:
+  replicas: 1
+  selector:
+    role: {{ .Values.nfsServerServiceName }}
+  template:
+    metadata:
+      labels:
+        role: {{ .Values.nfsServerServiceName }}
+    spec:
+      containers:
+      - name: {{ .Values.nfsServerServiceName }}
+        image: k8s.gcr.io/volume-nfs:0.8
```

Comment:
The image, image tag, and pull policy here should be configurable in the `values.yaml` file.

Reply:
This is what I intentionally left un-configurable in the NFS sub-chart. Because for the demo (as explained in my comment below), I don't want to expose these detailed nobs to the user.